content/articles/creating-a-simple-chat-app-in-react-using-the-chat-engine-api/index.md

```diff
@@ -0,0 +1,208 @@
+title: Creating a Simple Chat App in React.js using the chat engine API
+
+This article will teach the React.js way of creating a chat application using the chat engine API. 
+The learner will understand how the chat engine API works and how to use it to create a precise yet functional direct-messaging application.
+
+### Prerequisites
+- Primary React.js library mastery.  [The official documentation](https://reactjs.org/tutorial/tutorial.html) 
+- [Node.js](https://nodejs.org/en/download/) installed. 
+- [Node Package Manager (NPM)](https://docs.npmjs.com/cli/v6/commands/npm) know-how.
+
+### What chat engine API is?
+The chat engine is an API consumed to build chat services. 
+Approximating to other APIs that deliver user responses to the system, and sends back the system's responses to the user, this API works the same way, with better functionalities.
+It allows the hosting of these chats through the Rest API it lays out. The chat User Interface(UI), is created from the NPM components this tool comes with. See how the hustle is waved away for us already? 
+Furthermore, thanks to its server(s), the chat can as well be hosted from here(the server).
+This tool also saves you a lot of implementation time otherwise needed on many lines of code.
+Chat engine API can implement; 
+ - A group chat application.
+ - Direct messages chat application that would work like Facebook messenger .
+
+### Creating the Application
+This minute, let's begin creating the application.
+- On the terminal, cd to your project directory and run the following commands. 
+
+```bash
+npx create-react-app project-name
+```
+
+If you prefer using the Vite tool to create the React app, its step-by-step guide is taught in [this](https://www.section.io/engineering-education/creating-a-react-app-using-vite/) article. 
```

Comment:
What's Vite? some readers are just curious to read anything on the internet, let them understand everything you're discussing.